### PR TITLE
Siobhan, Mystagogue's Arctic Fox

### PR DESCRIPTION
**Pull request by TadJohnson00**
## About the PR

Adds Siobhan, the Mystagogue's arctic fox, as well as Rennie and Shivs toys.

**Media**

image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added arctic foxes to the game. Who knows, maybe you'll see one around!

### CODE REVIEW

**Comment by TadJohnson00** [comment]
Ayyy greentext first try

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/mobs.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- type: entity
+  name: Arctic Fox Siobhan Spawner
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/mobs.yml:2]: Isn't the other spawners' format something like `name: <animal> spawner` with `suffix: <pet name>`?
You're also missing generic arctic fox spawner

File: Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/animals.yml [hunk 2]
```diff
@@ -53,3 +53,69 @@
   - type: Grammar
     attributes:
       gender: epicene
+
+- type: entity
+  name: fox
+  parent: SimpleMobBase
+  id: MobArcticFox
+  description: They're a fox, of the arctic variety.
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/animals.yml]: This is lame, consider making some joke about them.

File: Resources/Prototypes/DeltaV/Entities/Objects/Fun/toys.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+- type: entity
+  parent: BasePlushie
+  id: ToyRenault
+  name: rennie toy
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Objects/Fun/toys.yml]:
```suggestion
  name: renault toy
```

File: Resources/Prototypes/DeltaV/Entities/Objects/Fun/toys.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    sound:
+      path: /Audio/Animals/fox_squeak.ogg
+  - type: MeleeWeapon
+    soundHit:
+      path: /Audio/Animals/fox_squeak.ogg
+
+- type: entity
+  parent: BasePlushie
+  id: ToySiobhan
+  name: shivs toy
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Objects/Fun/toys.yml]:
```suggestion
  name: siobhan toy
```

**Comment by DEATHB4DEFEAT** [comment]
The sprite reminded me of what I did to McGriff, so I did the same to this :)

mcg

mcg

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/mobs.yml (hunk 1 above)
Comment (TadJohnson00) [reply Resources/Prototypes/DeltaV/Entities/Markers/Spawners/mobs.yml:2]: Let me double check, and add a generic arctic fox spawner

File: Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/animals.yml (hunk 2 above)
Comment (TadJohnson00) [reply Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/animals.yml]: It was going to be a placeholder, I just forgot to change it

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/mobs.yml (hunk 1 above)
Comment (TadJohnson00) [reply Resources/Prototypes/DeltaV/Entities/Markers/Spawners/mobs.yml:2]: Nope, as far as I can see it follows a [Animal Type] [Name] Spawner (e.g. Fox Renault Spawner)

Comment (TadJohnson00) [reply Resources/Prototypes/DeltaV/Entities/Markers/Spawners/mobs.yml:2]: Scratch that, I'm not going to add a generic arctic fox spawner, there's no other generic animal spawners that I can see, with the exception of the random cat spawner and maybe one or two others.

**Comment by TadJohnson00** [comment]
Just to let you know, I was considering putting one or two bluespace crystals inside Shivs, but decided against it due to balance reasons... but then again, what sick fuck would butcher their own pet just for a measly couple of crystals?

**Comment by DebugOk** [comment]
Looks good enough, good luck mapping it though :clueless:

**Pull request merged**